File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// WebsiteAnalysisApis2 SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class WebsiteAnalysisApis2MakeContext
{
    public static function call(array $ctxmap, ?WebsiteAnalysisApis2Context $basectx): WebsiteAnalysisApis2Context
    {
        return new WebsiteAnalysisApis2Context($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# WebsiteAnalysisApis2 SDK utility: make_context
require_relative '../core/context'
module WebsiteAnalysisApis2Utilities
  MakeContext = ->(ctxmap, basectx) {
    WebsiteAnalysisApis2Context.new(ctxmap, basectx)
  }
end
